app/grandchallenge/reader_studies/models.py

```diff
@@ -1299,3 +1311,4 @@ def assign_permissions(self):
             self,
         )
         assign_perm(f"view_{self._meta.model_name}", self.creator, self)
+        assign_perm(f"change_{self._meta.model_name}", self.creator, self)
```

Comment:
Do we need to run a data migration to update the permissions of all answers?

Reply:
Then if answer is the only thing that is patchable then that is the only field that should be included in `HistoricalRecord`